test/Extensions.Tests/JavascriptSerializerTest.cs

```diff
@@ -0,0 +1,173 @@
+// MIT License.
+
+using System.Collections;
+using System.Collections.ObjectModel;
+using System.Web.Script.Serialization;
+using System.Web.UI.WebControls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace WebForms.Extensions.Tests;
+
+[TestClass]
+public class JavascriptSerializerTest
+{
+
+    [TestMethod]
+    public void JavaScriptSerializerConverterTest()
```

Comment:
can you include the output json as well? and have some pure deserialization tests as well?